connectivity/manifests/client-egress-toFQDNs-google.yaml

```
@@ -0,0 +1,31 @@
+apiVersion: cilium.io/v2
+kind: CiliumNetworkPolicy
+metadata:
+  namespace: cilium-test
+  name: client-egress-to-echo
```

This caused some confusing terminal output. We could check for duplicates after parsing, this might cause silent clobbering and let bugs slide.

```suggestion
  name: client-egress-to-fqdns-google
```

---

fixed, thanks!